scripts_generate/create_detection_data.py

```diff
@@ -247,21 +308,21 @@ def _strip_image(image):
 
 def _add_alphanumeric(image, shape, alpha, alpha_rgb, font_file):
     # Adjust alphanumeric size based on the shape it will be on
-    if shape == 'star':
+    if shape == "star":
         font_multiplier = 0.14
-    if shape == 'triangle':
+    if shape == "triangle":
```

Comment:
In the future this can be prob turned into a map (shape->shape_consts).

Reply:
Good idea.